package.json

```diff
@@ -561,5 +561,9 @@
     "*.{js,jsx,css,ts,tsx}": [
       "npx eslint --cache --fix --quiet"
     ]
+  },
+  "agentMetadata": {
+    "version": "2.0",
```

Comment:
fix version or match whole minor?

Reply:
This version string is defined by Azure Agent. We just follow it.